src/Composer/Plugin.php

```diff
@@ -151,10 +162,10 @@ protected function executeBltUpdate($version) {
       $this->io->write('<info>Creating BLT templated files...</info>');
       if ($this->isNewProject()) {
         // The BLT command will not work at this point because the .git dir doesn't exist yet.
-        $command = $this->getVendorPath() . '/acquia/blt/bin/blt internal:create-project --ansi';
+        $command = str_replace('\\', '/', PHP_BINDIR) . '/php ' . $this->getVendorPath() . '/acquia/blt/bin/blt internal:create-project --ansi --verbose';
```

Comment:
Why are you calling blt via the php binary here? Was something wrong with calling blt directly?

Additionally, why the hardcoded verbose option? This makes the output significantly more verbose and probably isn't desirable. Ideally this would inherit if someone uses `composer create-project --verbose`, but maybe that's a follow-up.

Reply:
Oh, this got in by mistake, I was trying to fix the windows problems. I'll remove it.